### ISSUE TITLE
Option "qp_iteration_limit" is unknown

### ISSUE THREAD

**Issue by odow** [ERGO-Code/HiGHS#1967]
Is there something special about `qp_iteration_limit` and `qp_nullspace_limit`?
```julia
julia> using HiGHS

julia> h = Highs_create()
Ptr{Nothing} @0x00007f8d4a76da00

julia> typeP = Ref{Cint}(0)
Base.RefValue{Int32}(0)

julia> Highs_getOptionType(h, "qp_iteration_limit", typeP)
ERROR:   getOptionIndex: Option "qp_iteration_limit" is unknown
-1

julia> Highs_getOptionType(h, "qp_nullspace_limit", typeP)
ERROR:   getOptionIndex: Option "qp_nullspace_limit" is unknown
-1
```

Raised by @sstroemer in https://github.com/jump-dev/HiGHS.jl/issues/231

**Comment by jajhall** [comment]
Strange, I'll look into this today, as there's probably a simple explanation and fix

**Comment by jajhall** [comment]
Calls to `getOptionType` are made for all options in [TestOptions.cpp](https://github.com/ERGO-Code/HiGHS/blob/master/check/TestOptions.cpp) so I can't see that there is a bug in HiGHS. My only thought is the version number. I introduced `qp_iteration_limit` and `qp_nullspace_limit` on 1 May 2024, so they will have been in releases from v1.7.1 (11 June 2024).

What version of HiGHS are you using, @sstroemer?

**Comment by sstroemer** [comment]
Thanks - that solved it! _(can't close this since it's not mine)_

If someone stumbles upon this: Doing `update HiGHS_jll` in package mode should be enough to fix this.

---

Just adding some context:

I'm using `HiGHS.jl`, version `v1.9.3` - however when calling `HiGHS` it showed version `1.7.0` (git hash: `50670fd4c`). I've recreated the environment on another computer, which now gives me (with the same `HiGHS.jl` version) `1.7.2`, where it works as you said. Why?

I did update `HiGHS.jl` some time ago, but it's compat entry currently lists `HiGHS_jll = "=1.5.1, =1.5.3, =1.6.0, =1.7.0, =1.7.1, =1.7.2"`, which left me at the old version of `HiGHS_jll.jl`. Interestingly enough, it seems like @odow was able to reproduce the bug in https://github.com/jump-dev/HiGHS.jl/issues/231, which I assume can only happen for a similar env with an old `HiGHS_jll.jl` - maybe that pitfall might be worth a note somewhere on the JuMP or MOI side of things (assuming there is the actual need to keep the old versions in the compat)?
